app/views/videos/show.html.haml

```diff
@@ -34,6 +34,6 @@
             %li.row
               .col-sm-2
                 %span Rating: #{review.rating} / 5
-                %p by #{link_to review.user.name, "#"}
+                %p by #{link_to review.user.name, review.user}
               .col-sm-8
                 %p #{review.body}
```

Comment:
I used interpolation because of the "by" on the line which is html. I was getting haml errors without the interpolation